week-6/Homework/mandatory/3-slideshow/slideshow.js

```diff
@@ -1 +1,70 @@
 // Write your code here
+const autoPreviusButton = document.getElementById("auto-backward");
+const previusButton = document.getElementById("backward");
+const pauseButton = document.getElementById("pause");
+const nextButton = document.getElementById("forward");
+const autoNextsButton = document.getElementById("auto-forward");
+let imageSource = document.getElementById("pic");
+let images = [
+  "pictures/charles-postiaux-Q6UehpkBSnQ-unsplash.jpg",
+  "pictures/chris-karidis-nnzkZNYWHaU-unsplash.jpg",
+  "pictures/ferdinand-stohr-PeFk7fzxTdk-unsplash.jpg",
+  "pictures/joshua-ang-Gf_KqXHU-PY-unsplash.jpg",
+  "pictures/wael-hneini-QJKEa9n3yN8-unsplash.jpg",
+];
+let i = 0;
+let autoForward;
+let autoGetBack;
+
+function imageChange() {
+  autoNextsButton.addEventListener("click", () => {
+    clearInterval(autoGetBack);
+    clearInterval(autoForward);
+    pauseButton.style.color = "black";
+    autoForward = setInterval(() => {
+      imageSource.src = images[i++];
+      if (i == images.length) {
+        i = 0;
+      }
+    }, 3000);
+  });
+
+  autoPreviusButton.addEventListener("click", () => {
+    clearInterval(autoForward);
+    clearInterval(autoGetBack);
+    pauseButton.style.color = "black";
```

Comment:
This block of code has been repeated with some minor changes - you may want to consider abstracting into a separate function so that you can reuse it